rules/gfb.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var message = require('lambda-cfn').message;
+var util = require('util');
+
+module.exports.config = {
+    name: 'gfb',
+    sourcePath: 'rules/gfb.js',
+    gatewayRule: {
+        method: 'POST',
+        apiKey: false
+    }
+};
+
+module.exports.fn = function (event, callback) {
+    if (event.zen != undefined) {
+        var ping = 'GitHub ping event received';
+        return callback(null, ping);
+    } else {
+        console.log('event ', event);
+        if (event.action === 'labeled'){
+            var label = event.label;
+            if (label.name === 'good-starter-bug') {
```

Comment:
Since this rule is fired every time an issue is `assigned, unassigned, labeled, unlabeled, opened, edited, milestoned, demilestoned, closed, or reopened`, will this rule end up spamming the slack channel if it is updated heavily?